language/move-core/types/src/transaction_argument.rs

```diff
@@ -11,7 +11,7 @@ pub enum TransactionArgument {
     U64(u64),
     U128(u128),
     Address(AccountAddress),
-    U8Vector(Vec<u8>),
+    U8Vector(#[serde(with = "serde_bytes")] Vec<u8>),
```

Comment:
should we rename this to BYTES as well?

Reply:
I think it would make a lot of sense but I cannot speak for Move specialists (@vgao1996 @sblackshear @dariorussi what do you think?)

Reply:
note: I'm going to land this in the meantime, but we can always iterate in a next PR.